spec/shop_spec.rb

```diff
@@ -27,8 +27,20 @@
   describe "#all" do
     rows = DB[:shops].all
     it 'should return all rows from the shops table' do
-      shop = Shop.new "cafe"
+      shop = Shop.new
       expect(Shop.all).to eq(rows)
     end
   end
+
+  describe "#find" do
+    shop = Shop.new({name: 'RED', description: 'red apples'})
+    v = Shop.find(15)
```

Comment:
as `shops` table has an auto-increment ID, finding for a fixed ID (e.g. 15) might break the specs. I'd recommend changing it to a more dynamic approach like calling an `id` method on `shop` instance.

also, you can replace `v` to a more meaningful variable like `result.`